tests/test_tztrout.py

```diff
@@ -0,0 +1,433 @@
+import datetime
+
+import pytest
+import tztrout
+from mock import patch
+
+
+class FakeDateTime(datetime.datetime):
+    "A datetime replacement that lets you set utcnow()"
+
+    @classmethod
+    def utcnow(cls, *args, **kwargs):
+        if hasattr(cls, 'dt'):
+            return cls.dt
+        raise NotImplementedError(
+            'use FakeDateTime.set_utcnow(datetime) first'
+        )
+
+    @classmethod
+    def set_utcnow(cls, dt):
+        cls.dt = dt
+
+
+class TestTZTrout:
+    def assert_only_one_tz(self, ids, tz_name, tz_names):
+        """ Assert that a given set of timezone ids only matches one tz name
+         in a given set of tz names
+         """
+        tz_names_copy = tz_names[:]
+        assert tz_name in tz_names_copy
+        tz_names_copy.remove(tz_name)
+        assert set(tztrout.tz_ids_for_tz_name(tz_name)) & set(ids)
+        for other_name in tz_names_copy:
+            assert not (set(tztrout.tz_ids_for_tz_name(other_name)) & set(ids))
+
+    @pytest.fixture
+    def us_ca_tz_names(self):
+        return ['PT', 'MT', 'CT', 'ET', 'AT']
+
+    @pytest.fixture
+    def au_tz_names(self):
+        return ['AWT', 'ACT', 'AET']
```

Comment:
These might as well be constants, they don't need to be fixtures.

Reply:
done in 30e01b7